### DISCUSSION TITLE
v0.10.0-beta.1

### DISCUSSION THREAD

**Discussion by Samuel12321** [ShankarBUS/ModernFlyouts#850] [General]
### Please Note that this Beta is published as framework dependent, this means you will also need to install the .NET 6 runtime that matches your system architecture. 
This is a one off situation, future releases will be published self contained as usual, this was just done for this beta so it could be released while we are still working on a fix for a self-contained bug.

### Bug Fixes
- Fixed #470 - Bug: Flyouts doesn't appear while holding any mouse button
- Fixed #592 - Bug: Switching off 'Enable Audio Flyout' causes crash
- Fixed a bug where dragging the flyout breaks the placement mode setting
- Fixed #584 - Bug: Volume flyout pops up when changing keyboard layout
- Fixed #311 - Bug: Use mouse change volume to 0, the sound icon not become mute
- Fixed a bug where the flyout's location was not restored on startup
### What's New
- #### ARM64 Support #482 - We now Natively support ARM64, the MSIX bundle includes an ARM64 build.
- Resolved #392 - Feature Request: Hide Timeline bar when not supported by media app. The media control layout has been updated. Many people complained that the timeline slider was useless in many cases and taking up a lot of space. In this release the slider will only be shown when you click the timeline info button.
- Initial implementation for #102. As for now, you can use the brightness module settings page to control the brightness. Hotkey support will be added in the next version.
- Resolved #415 - Feature Request: Click an empty space to hide the flyout.
- Resolved #460 - Feature Request: Realign text block to album art.
- Resolved #60 - [Enhancement] The side arrows overlap with the Title and the Thumbnail.
- Translation Updates

### Please let us know what you think of the changes in the discussion for this release or on the discord chat.
.

| Media session (Timeline unsupported) | Media session (Timeline supported, Collapsed) | Media session (Timeline supported, Expanded) |
|---|---|---|
| ![image](https://user-images.githubusercontent.com/63704247/128132299-bc07f995-3e03-4899-99e6-27883fa0e1c2.png) | ![image](https://user-images.githubusercontent.com/63704247/128132391-ad99c7e1-cd8a-4f0a-90c0-ecd3060b60fc.png) | ![image](https://user-images.githubusercontent.com/63704247/128132393-ced39da0-91c1-43dc-af68-106e53c52199.png) |

| Media session (Horizontal scroll button) | Media session (Vertical scroll button) |
|---|---|
| ![image](https://user-images.githubusercontent.com/63704247/128132478-6ab74bd0-3d2f-4f92-af6c-a505b4d5f1fe.png) | ![image](https://user-images.githubusercontent.com/63704247/128132598-3f5160b0-cbda-4245-96a0-51ba675d042c.png) |

**New multi monitor brightness controls in the Brightness module settings page**

![image](https://user-images.githubusercontent.com/63704247/128135516-f72f95f0-e83b-4318-bd71-9e72979603b1.png)

### Other:
Remember to install the certificate in the trusted root authority under "local machine". As this is a pre-release i haven't signed it via the MS Store.

<hr /><em>This discussion was created from the release <a href='https://github.com/ModernFlyouts-Community/ModernFlyouts/releases/tag/v0.10'>v0.10.0-beta.1</a>.</em>

**Comment by J-HS-L** [comment]
Is this far off from being a release in winget or the store?

**Reply by Samuel12321** [reply]
Hi,
Not sure of timeline yet, the team has been very busy. There are still a few stability issues.
There is a chance that we may skip this release and jump straight to 0.11 which will be a major rebuild of the ui to align with Windows 11 styles and add new features, such as tray flyouts.

Will give an update once the team has decided on the timeline.

**Reply by J-HS-L** [reply]
~Hmm, no email about this...~ 
~Glad I was perusing some browsing history, else I wouldn't have known.~
~>! I need to hurry-up & stop relying [**even more**] on email for notifications.~ 
~>! As we all know, email is sooo broken thanks to abuse!~ 

OK, thank you for the update, look forward to hearing plans soon-ish.

Best Regards!

**Comment by FoxBlocks** [comment]
Is it possible to back up my settings from the stable version before switching to the beta? If so, where can I find the file storing them?
EDIT: Nevermind, I found [this](https://github.com/ModernFlyouts-Community/ModernFlyouts/issues/290#issuecomment-739441390).

**Reply by Samuel12321** [reply]
Coming in 0.11

**Comment by gnpaone** [comment]
Is it possible to add flyout for keyboard backlight levels, touchpad enable/disable, camera privacy?

**Reply by Samuel12321** [reply]
Keyboard backlight -no firmware controlled
Touchpad - maybe
Camera- coming soon with 0.11

**Comment by Samuel12321** [comment]
Closing thread as no longer relevant.
